src/platform/forms-system/test/js/review/SubmitController.unit.spec.jsx

```diff
@@ -185,29 +193,34 @@ describe('Schemaform review: SubmitController', () => {
     tree.unmount();
   });
 
-  it('should not submit when invalid', () => {
-    const form = createForm();
-    // Form with missing rquired field
+  it('should not submit when invalid data is entered', () => {
```

Comment:
Previously, this test was succeeding because the `SubmitController` didn't submit the form and recorded that it was submitted. I discovered, however, that this happened because the required privacy agreement wasn't _actually_ `true`. I changed this test up to ensure that the failure we expect is triggered from a validation error.

It's not technically a part of the `inProgressFormId` work, but I needed to mimic the validation error to test the Sentry call anyhow, so I updated this while I was at it.